d3a_api_client/cli.py

```diff
@@ -72,30 +73,51 @@ def main(log_level):
                   ', '.join(_setup_modules)))
 @click.option('-u', '--username', default=None, type=str, help="D3A username")
 @click.option('-p', '--password', default=None, type=str, help="D3A password")
-@click.option('-d', '--domain-name', default=DEFAULT_DOMAIN_NAME,
+@click.option('-d', '--domain-name', default=None,
               type=str, help="D3A domain name")
-@click.option('-w', '--web-socket', default=DEFAULT_WEBSOCKET_DOMAIN,
+@click.option('-w', '--web-socket', default=None,
               type=str, help="D3A websocket URI")
-@click.option('-i', '--simulation-config-path', type=str,
+@click.option('-i', '--simulation-config-path', type=str, default=None,
               help="Path to simulation config file (accept absolute and relative path)")
+@click.option('-s', '--simulation-id', type=str, default=None,
+              help="Simulation id")
 @click.option('--run-on-redis', is_flag=True, default=False,
               help="Start the client using the Redis API")
 def run(base_setup_path, setup_module_name, username, password, domain_name, web_socket,
-        simulation_config_path, run_on_redis, **kwargs):
+        simulation_config_path, simulation_id, run_on_redis, **kwargs):
     if username is not None:
         os.environ["API_CLIENT_USERNAME"] = username
     if password is not None:
         os.environ["API_CLIENT_PASSWORD"] = password
-    os.environ["API_CLIENT_DOMAIN_NAME"] = domain_name
-    os.environ["API_CLIENT_WEBSOCKET_DOMAIN_NAME"] = web_socket
+
     os.environ["API_CLIENT_RUN_ON_REDIS"] = "true" if run_on_redis else "false"
-    os.environ["SIMULATION_CONFIG_FILE_PATH"] = create_simulation_config_path(
-        base_setup_path, simulation_config_path
-    ) if run_on_redis is False else ""
+    if simulation_config_path is not None and not run_on_redis:
+        config_file_path = create_simulation_config_path(base_setup_path, simulation_config_path)
+        config = read_simulation_config_file(config_file_path)
+        os.environ["API_CLIENT_DOMAIN_NAME"] = config["domain_name"]
+        os.environ["API_CLIENT_WEBSOCKET_DOMAIN_NAME"] = config["web_socket_domain_name"]
+        os.environ["API_CLIENT_SIMULATION_ID"] = config["uuid"]
+    else:
+        os.environ["API_CLIENT_DOMAIN_NAME"] = domain_name \
+            if domain_name is not None else domain_name_from_env()
```

Comment:
This won't break if `domain_name == ''`. If that should be avoided, I'd recommend something like:
```suggestion
        os.environ["API_CLIENT_DOMAIN_NAME"] = (
            domain_name if domain_name else domain_name_from_env())
```